app/src/main/java/net/skyscanner/backpack/demo/stories/Calendar2Story.kt

```diff
@@ -83,40 +85,40 @@ class Calendar2Story : Story() {
         .toSet()
     } else emptySet()
 
-    var info = emptyMap<LocalDate, CalendarParams.Info>()
+    var info = emptyMap<LocalDate, CellInfo>()
 
     if (type.disableWeekends) {
       info = range
-        .toMap { CalendarParams.Info(status = CalendarParams.Status.Disabled) }
+        .toMap { CellInfo(status = CellStatus.Disabled) }
         .filter { it.key.dayOfWeek == DayOfWeek.SATURDAY || it.key.dayOfWeek == DayOfWeek.SUNDAY }
     } else if (type.labels) {
       info = range.toMap {
         val price = random.nextInt(100)
-        CalendarParams.Info(
+        CellInfo(
           label = when (price) {
             in 0..25 -> "–"
             else -> "£$price"
           },
           status = when (price) {
-            in 0..25 -> CalendarParams.Status.Empty
-            in 25..50 -> CalendarParams.Status.Positive
-            in 50..75 -> CalendarParams.Status.Neutral
-            in 75..100 -> CalendarParams.Status.Negative
-            else -> CalendarParams.Status.Empty
+            in 0..25 -> CellStatus.Empty
+            in 25..50 -> CellStatus.Positive
+            in 50..75 -> CellStatus.Neutral
+            in 75..100 -> CellStatus.Negative
+            else -> CellStatus.Empty
           }
         )
       }
     } else if (type.colored) {
       val random = Random(0)
       info = range.toMap {
         val price = random.nextInt(100)
-        CalendarParams.Info(
+        CellInfo(
           status = when (price) {
-            in 0..25 -> CalendarParams.Status.Empty
-            in 25..50 -> CalendarParams.Status.Positive
-            in 50..75 -> CalendarParams.Status.Neutral
-            in 75..100 -> CalendarParams.Status.Negative
-            else -> CalendarParams.Status.Empty
+            in 0..25 -> CellStatus.Empty
```

Comment:
maybe these could be defined as constants (especially as we have to define them twice as well)